### PR TITLE
repackage of FR

### PR DESCRIPTION
**Pull request by ilaflamme-va**

### CODE REVIEW

**Comment by jblefkove-va** [comment]
Lots of static imports removed?

**Comment by ilaflamme-va** [comment]
Yea, the static import was replaced with the whole class :(
Intelij why you do this :-1:

File: argonaut-dstu2-examples/checkstyle-suppressions.xml [hunk 1]
```diff
@@ -5,6 +5,7 @@
   <!--
     Ignore a long realistic example strings.
    -->
+  <suppress checks="LineLength" files="SwaggerMedication.java" lines="89" />
```
Comment (jblefkove-va) [inline argonaut-dstu2-examples/checkstyle-suppressions.xml:8]: These line-specific suppressions are brittle. Can we split the offending line in two instead?

Comment (ilaflamme-va) [reply argonaut-dstu2-examples/checkstyle-suppressions.xml:8]: Yeah, that was in several places that I had to change. I can go break them up and remove the suppressions.

File: argonaut-dstu2-examples/src/main/java/gov/va/api/health/dstu2/api/swaggerexamples/SwaggerAllergyIntolerance.java [hunk 2]
```diff
@@ -1,8 +1,7 @@
-package gov.va.api.health.argonaut.api.swaggerexamples;
```
Comment (jblefkove-va) [inline argonaut-dstu2-examples/src/main/java/gov/va/api/health/dstu2/api/swaggerexamples/SwaggerAllergyIntolerance.java:1]: Don't forget to update the packages for the inline references to the swagger examples

Comment (ilaflamme-va) [reply argonaut-dstu2-examples/src/main/java/gov/va/api/health/dstu2/api/swaggerexamples/SwaggerAllergyIntolerance.java:1]: Good, catch. Missed lots of those.

File: argonaut-dstu2/src/main/java/gov/va/api/health/dstu2/api/resources/Patient.java [hunk 3]
```diff
@@ -47,9 +46,9 @@
 @AllArgsConstructor
 @JsonAutoDetect(fieldVisibility = JsonAutoDetect.Visibility.ANY)
 @Schema(
-    description = "http://www.fhir.org/guides/argonaut/r2/StructureDefinition-argo-patient.html",
+    description = "http://www.fhir.org/guides/dstu2/r2/StructureDefinition-argo-patient.html",
```
Comment (jhulbert-va) [inline argonaut-dstu2/src/main/java/gov/va/api/health/dstu2/api/resources/Patient.java:49]: Was this intentional or sed gone wrong?

**Review by bschofield-va — CHANGES_REQUESTED** [review]
Sample injection is going to break now that the examples are concatenated strings, e.g.,
```
example =
         "${argonaut.medicationOrder:gov.va.api.health.argonaut.api.swaggerexamples.SwaggerMedicationOrder#medicationOrder}")		         "${dstu2.medicationOrder:gov.va.api.health.dstu2.api.swaggerexamples"
             + ".SwaggerMedicationOrder#medicationOrder}
```

related: https://github.com/department-of-veterans-affairs/swagger-examples-maven-plugin/blob/master/src/main/java/gov/va/plugin/maven/swagger/ExampleInjector.java#L29

**Comment by bschofield-va** [comment]
> Yea, the static import was replaced with the whole class
> Intelij why you do this 👎

So does this mean you're restoring it to the way it was?

**Comment by jblefkove-va** [comment]
Doesn't sample injection handle concatenated strings? We're already splitting long example strings in some places.

File: us-core-r4/src/main/java/gov/va/api/health/r4/api/resources/Condition.java [hunk 4]
```diff
@@ -48,8 +47,7 @@
     description =
         "https://build.fhir.org/ig/HL7/US-Core-R4/StructureDefinition-us-core-condition.html",
     example =
-        "${uscorer4.condition:gov.va.api.health."
-            + "uscorer4.api.swaggerexamples.SwaggerCondition#condition}")
+        "${r4.condition:gov.va.api.health." + "r4.api.swaggerexamples.SwaggerCondition#condition}")
```
Comment (jhulbert-va) [inline us-core-r4/src/main/java/gov/va/api/health/r4/api/resources/Condition.java:50]: Oof

File: us-core-r4/src/main/java/gov/va/api/health/r4/api/resources/Location.java [hunk 5]
```diff
@@ -43,8 +42,7 @@
     description =
         "https://build.fhir.org/ig/HL7/US-Core-R4/StructureDefinition-us-core-location.html",
     example =
-        "${uscorer4.location:gov.va.api.health.uscorer4.api."
-            + "swaggerexamples.SwaggerLocation#location}")
+        "${r4.location:gov.va.api.health.r4.api." + "swaggerexamples.SwaggerLocation#location}")
```
Comment (jhulbert-va) [inline us-core-r4/src/main/java/gov/va/api/health/r4/api/resources/Location.java:45]: Formatter

File: us-core-r4/src/main/java/gov/va/api/health/r4/api/resources/Procedure.java [hunk 6]
```diff
@@ -47,8 +46,7 @@
     description =
         "https://build.fhir.org/ig/HL7/US-Core-R4/StructureDefinition-us-core-procedure.html",
     example =
-        "${uscorer4.procedure:gov.va.api.health."
-            + "uscorer4.api.swaggerexamples.SwaggerProcedure#procedure}")
+        "${r4.procedure:gov.va.api.health." + "r4.api.swaggerexamples.SwaggerProcedure#procedure}")
```
Comment (jhulbert-va) [inline us-core-r4/src/main/java/gov/va/api/health/r4/api/resources/Procedure.java:49]: :disappointed:

**Comment by ilaflamme-va** [comment]
> > Yea, the static import was replaced with the whole class
> > Intelij why you do this -1
> 
> So does this mean you're restoring it to the way it was?

I fix

File: argonaut-dstu2/src/main/java/gov/va/api/health/dstu2/api/resources/Patient.java [hunk 7]
```diff
@@ -153,8 +152,8 @@ private boolean isValidRaceExtension() {
   @Schema(
       name = "PatientBundle",
       example =
```
Comment (eclendenning-va) [inline argonaut-dstu2/src/main/java/gov/va/api/health/dstu2/api/resources/Patient.java:154]: Formatter

File: argonaut-dstu2/pom.xml [hunk 8]
```diff
@@ -7,8 +7,8 @@
     <relativePath/>
   </parent>
   <modelVersion>4.0.0</modelVersion>
-  <artifactId>r4</artifactId>
-  <version>4.0.19-SNAPSHOT</version>
+  <artifactId>argonaut-dstu2</artifactId>
```
Comment (jblefkove-va) [inline argonaut-dstu2/pom.xml:10]: Is this GitHub getting confused?

Comment (jhulbert-va) [reply argonaut-dstu2/pom.xml:10]: Yeah. I've got it locally and everything looks good.

File: argonaut-dstu2/src/test/java/gov/va/api/health/dstu2/api/resources/DiagnosticReportTest.java [hunk 9]
```diff
@@ -71,17 +71,18 @@ public void relatedGroups() {
 
   @Test
   public void validationFailsGivenBadCategory() {
-    assertThat(violationsOf(data.diagnosticReport().category(data.codeableConcept()))).isNotEmpty();
+    Assertions.assertThat(violationsOf(data.diagnosticReport().category(data.codeableConcept())))
```
Comment (jblefkove-va) [inline argonaut-dstu2/src/test/java/gov/va/api/health/dstu2/api/resources/DiagnosticReportTest.java:74]: static imports

File: argonaut-dstu2/src/test/java/gov/va/api/health/dstu2/api/resources/ObservationTest.java [hunk 10]
```diff
@@ -81,17 +81,18 @@ public void relatedGroups() {
 
   @Test
   public void validationFailsGivenBadCategory() {
-    assertThat(violationsOf(data.observation().category(data.codeableConcept()))).isNotEmpty();
+    Assertions.assertThat(violationsOf(data.observation().category(data.codeableConcept())))
```
Comment (jblefkove-va) [inline argonaut-dstu2/src/test/java/gov/va/api/health/dstu2/api/resources/ObservationTest.java:84]: more imports

File: us-core-r4/checkstyle-suppressions.xml [hunk 11]
```diff
@@ -5,9 +5,10 @@
   <!--
     The following resources have certain fields that are reported
     as data absent reason extensions that is required to have a _
-    prefix by the us-core-r4 specification.
+    prefix by the Data Query specification.
```
Comment (jblefkove-va) [inline us-core-r4/checkstyle-suppressions.xml:8]: I think the old comment was more accurate?

File: us-core-r4/src/test/java/gov/va/api/health/r4/api/resources/DiagnosticReportTest.java [hunk 12]
```diff
@@ -75,7 +75,7 @@ void relatedGroups() {
 
   @Test
   void validationFailsGivenBadCategorySystem() {
-    assertThat(
+    Assertions.assertThat(
```
Comment (jblefkove-va) [inline us-core-r4/src/test/java/gov/va/api/health/r4/api/resources/DiagnosticReportTest.java]: imports

File: us-core-r4/src/test/java/gov/va/api/health/r4/api/resources/EncounterTest.java [hunk 13]
```diff
@@ -60,7 +60,8 @@ public void encounter() {
 
   @Test
   public void validationFailsGivenBadIdentifier() {
-    assertThat(violationsOf(data.encounter().identifier(singletonList(data.identifier()))))
+    Assertions.assertThat(
```
Comment (jblefkove-va) [inline us-core-r4/src/test/java/gov/va/api/health/r4/api/resources/EncounterTest.java]: imports

**Review by jblefkove-va — APPROVED** [review]

**Review by jhulbert-va — APPROVED** [review]
:shipit:

**Review by jhulbert-va — APPROVED** [review]

**Pull request merged**